lms/templates/learner_dashboard/course_entitlement.underscore

```diff
@@ -0,0 +1,26 @@
+<div id="change-session-<%-entitlementUUID%>" class="message is-shown">
+    <div class="action-header">
+        <% if (currentSessionId) { %>
+            <%- gettext('To change your session or leave your current session, please select from the following:')%>
+        <% } else { %>
+            <%- gettext('In order to view the course you must select a session:')%>
+        <% } %>
+    </div>
+    <div class="action-controls">
+        <select class="session-select">
+            <% _.each(availableSessions, function(session) { %>
+                <option data-session_id="<%-session.session_id%>">
+                    <%-session.session_dates%>
+                    <% if (session.session_id == currentSessionId) { %>
+                        <%- gettext('  - Currently Enrolled')%>
+                    <% } else if (session.enrollment_end){ %>
+                        <%- gettext(' (Enroll by ') + session.enrollment_end + ')'%>
+                    <% } %>
+                </option>
+            <% }) %>
+            <option disabled><%- gettext('More sessions coming soon')%></option>
+            <% if (currentSessionId){%> <option><%- gettext('Leave current session and decide later.')%></option><%}%>
+        </select>
+        <button class="enroll-btn-initial"> <%- gettext('Enroll In Session') %></button>
```

Comment:
Nit: I'm not sure if it's intentional, but there's an extra space before the gettext call.